### DISCUSSION TITLE
Max-p-compact regions availability

### DISCUSSION THREAD

**Discussion by samueltoro7** [pysal/spopt#394] [Q&A]
Hi Pysal community,

I am wondering if the algorithm for the max-p-compact regions problem is already implemented in the Pysal library or where I can get it.
I read the paper published by Xin Feng, Sergio Rey and Ran Wei and I found really interesting and useful.

I look forward for your reply!

Cheers
Samuel

**Comment by knaaptime** [comment]
hi all, The compact max-p algorithm used in that paper was implemented by @xf37 and I don't think a PR into the library was ever finalized. I'm not sure what the plans are at the moment
